src/modifiers/handleInlineStyle.js

```diff
@@ -14,6 +14,10 @@ const handleChange = (editorState, line, whitelist) => {
         do {
           matchArr = re.exec(line);
           if (matchArr) {
+            if (matchArr[0][0] === " ") {
```

Comment:
I think this needs to check for any whitespace char with a regex (`\s`), like the regex for inline style matchers? Using a tab or a non-breaking space will break this removal.